lib/src/errors.dart

```diff
@@ -304,6 +304,12 @@ class SemanticError extends IssueType {
           (args) =>
               'All primitives must have the same number of morph targets.');
 
+  static final SemanticError meshPrimitivesUnequalJointsCount =
+      new SemanticError._(
+          'MESH_PRIMITIVES_UNEQUAL_JOINTS_COUNT',
+          (args) => "All primitives must contain the same number of 'JOINTS' "
+              "and 'WEIGHTS' attribute sets.");
```

Comment:
I think I like the idea of error or warning here, but is this mentioned in the spec?

Reply:
https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#node

Reply:
The spec requires each primitive have `JOINTS_0` and `WEIGHTS_0`, but I see nothing to say that one primitive could have `JOINTS_1` in addition to `JOINTS_0` and not other primitives. The spec would lead me to think that is valid, while this error would lead me to think it is not.

Reply:
I don't have a data-driven opinion on this. We haven't yet seen models with more than 4 influences per vertex. 

What do you think?

Reply:
About whether it should be allowed, or whether the validator should say something about it?

I haven't seen any examples either. I'd prefer that primitives be more consistent than not, but elsewhere we've asserted [primitives could have different attribute sets](https://github.com/KhronosGroup/glTF/issues/1090) without discussing skinning specifically...

The message is sort of saying one thing and testing another, but tentatively, I'd lean toward  making the error code something more generic (`MESH_PRIMITIVES_INVALID_JOINTS_COUNT`) and then we can change either the behavior or the test or the message text depending on what we decide in the future.

Reply:
> I'd lean toward making the error code something more generic 

Agree. 

@zellski do you have preference on this?

Reply:
Thanks for checking -- no, no strong preference. I feel mildly irked by the thought of completely heterogenous primitives in a mesh, but I have no rational basis for that irritation. When we implement animation at Facebook, I expect we'll treat primitives in a mesh as almost completely unrelated to one another, so no strong "day job" preference either.

Reply:
Considering that skinned mesh transform should be ignored, all primitives of the skinned mesh must have joints. Whether they have to have the same number of influences per vertex is a different question. 

I'll update the code and tests.

Reply:
Turns out we have a separate issue for skinned primitives without joints:
https://github.com/KhronosGroup/glTF-Validator/blob/d376c03d0764a27401d8867e640b9f91908f5686/lib/src/base/node.dart#L198-L201

So I propose to replace "must" with "should" in the description of `MESH_PRIMITIVES_UNEQUAL_JOINTS_COUNT` (while keeping error code) and to reduce its severity to `Warning` or `Information`.

Reply:
Sounds good 👍